webrtc.html

```diff
@@ -11344,9 +11320,33 @@ <h3>Perfect Negotiation Example</h3>
       </ol>
       <p>Together, they manage signaling for the rest of the application in a
       manner that doesn't deadlock. The example assumes a
-      <var>polite</var> boolean variable indicating the designated role:</p>
+      <code>polite</code> boolean variable indicating the designated role:</p>
       <pre class="example highlight">
-// The perfect negotiation logic, separated from the rest of an application ---
+const signaling = new SignalingChannel(); // handles JSON.stringify/parse
+const constraints = {audio: true, video: true};
+const configuration = {iceServers: [{urls: 'stun:stun.example.org'}]};
+const pc = new RTCPeerConnection(configuration);
+
+// call start() anytime on either end to add camera and microphone to connection
+async function start() {
+  try {
+    const stream = await navigator.mediaDevices.getUserMedia(constraints);
+    for (const track of stream.getTracks()) {
+      pc.addTrack(track, stream);
+    }
+    selfView.srcObject = stream;
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+pc.ontrack = ({streams}) =&gt; {
+  if (remoteView.srcObject) return;
+  remoteView.srcObject = streams[0];
+};
+
+// - The perfect negotiation logic, separated from the rest of the application ---
```

Comment:
are the dashes serving some purpose, or are they a distraction?

Reply:
They form a separator, to illustrate separation of the perfect negotiation logic below (the remaining code) from the app logic above.

Unlike other comments, it does not apply only to the following lines/function. Do we have other ways to illustrate that?